src/index.ts

```diff
@@ -10,13 +11,13 @@ const oneDay = oneHour * 24
 const oneYear = oneDay * 365
 const immutableCacheTime = oneYear
 // The not found time should be small to help in the following case:
-// 1. Somebody requests an exact version that doesn't exists, CloudFront caches the 404 response;
+// 1. Somebody requests an exact version that doesn't exist, CloudFront caches the 404 response;
 // 2. The exact version is published and the inexact endpoint redirects to it.
 const notFoundBrowserCacheTime = oneHour
 const tempNotFoundCdnCacheTime = oneMinute * 5
-const tempRedirectBrowserCacheTime = oneDay * 7
-const tempRedirectCdnCacheTime = oneHour
-const monitoringBrowserCacheTime = tempRedirectBrowserCacheTime
+const tempAliasBrowserCacheTime = oneDay * 7
```

Comment:
Does this imply that if someone requests `/package/v3` and gets the current latest version (for example 3.3.3) and we release 3.3.4, it might take 7 days for this client to get it? If yes, was this the case with the redirects, too?

Reply:
Yes and yes